cafebabel/templates/articles/macros.html

```diff
@@ -17,13 +17,16 @@
 {%- endmacro %}
 
 {%- macro tags_fields(tags, article=None) -%}
-  <div>
+  <div class=tags>
     <label for=tag-1>Tags</label>
-    <input list=tags name=tag-1 {% if article and article.tags and article.tags[0] %}value={{ article.tags[0].name }}{% endif %}>
-    <input list=tags name=tag-2 {% if article and article.tags and article.tags[1] %}value={{ article.tags[1].name }}{% endif %}>
-    <input list=tags name=tag-3 {% if article and article.tags and article.tags[2] %}value={{ article.tags[2].name }}{% endif %}>
-    <input list=tags name=tag-4 {% if article and article.tags and article.tags[3] %}value={{ article.tags[3].name }}{% endif %}>
-    <input list=tags name=tag-5 {% if article and article.tags and article.tags[4] %}value={{ article.tags[4].name }}{% endif %}>
+    <div class=tag-container><input name=tag-new><button class=add></button></div>
+    {% if article and article.tags %}
```

Comment:
`{% if … article.tags %}`: does it still work when creating an article or editing an article with no tag, or will these fields not appear at all?